src/software/thunderscope/thunderscope_main.py

```diff
@@ -38,9 +38,24 @@
 
 if __name__ == "__main__":
 
-    logging.getLogger().setLevel(logging.INFO)
+    ###########################################################################
+    #                              Logging                                    #
+    ###########################################################################
+    #
+    # Setup logging
+    logger = logging.getLogger("Thunderscope")
+    logger.setLevel(logging.DEBUG)
```

Comment:
do we wanna change from logging.INFO to logging.DEBUG?

Reply:
could by a command line arg